modules/node/handler/page_builder.e

```diff
@@ -0,0 +1,70 @@
+note
+	description: "Generic Page Builder Interface"
+	date: "$Date$"
+	revision: "$Revision$"
+
+deferred class
+	PAGE_BUILDER [G->CMS_NODE]
```

Comment:
what about
NODE_PAGINATION_BUILDER  for the name
the current name may get user confused , by thinking this is related to CMS_PAGE ...